src/intersection-observer.js

```diff
@@ -103,9 +85,9 @@ export function getIntersectionChangeEntry(element, owner, viewport) {
 
   return /** @type {!IntersectionObserverEntry} */ ({
     time: Date.now(),
-    rootBounds: DomRectFromLayoutRect(rootBounds),
```

Comment:
As discussed with @lannka, I changed Layout Rect to have same value with Dom Rect (add x and y value)s, so we no longer need to convert from Dom Rect to Layout Rect